ethcore/sync/src/chain/supplier.rs

```diff
@@ -266,6 +282,10 @@ impl SyncSupplier {
 				added_receipts += receipts_bytes.len();
 				added_headers += 1;
 				if added_receipts > MAX_RECEIPTS_TO_SEND { break; }
```

Comment:
Can we get rid of this? It seems redudant now.

Reply:
Acutally, I kept `MAX_RECEIPTS_TO_SEND` which as you pointed is already in bytes (cf. https://github.com/paritytech/parity-ethereum/pull/10042#discussion_r240349764).

Reply:
Right, by do you know why do we have this limit in the first place? Maybe let's use soft limit instead?

Reply:
Yeah I don't know either, let me do that.